Makefile

```diff
@@ -146,6 +146,9 @@ functionaltest: | nvim
 functionaltest-lua: | nvim
 	+$(BUILD_TOOL) -C build functionaltest-lua
 
+stylua:
+	stylua --check runtime/
+
```

Comment:
On the other hand, `lualint` and `clint` are not optional, either?

Reply:
`lualint` is optional, just that the check is done in Cmakelist.txt instead (don't ask why lol). `clint` is our own tool which is always available, so that shouldn't be a problem.